Gemfile

```diff
@@ -2,18 +2,19 @@ source 'https://rubygems.org'
 
 gem 'aws-sdk'
 gem 'carrierwave', '~> 0.10.0'
-gem 'carrierwave-mongoid', '~> 0.8.1', require: 'carrierwave/mongoid'
-gem 'gds-sso', '~> 11.2'
+gem 'carrierwave-mongoid', '~> 0.10.0', require: 'carrierwave/mongoid'
+gem 'gds-sso', '~> 13.2'
 gem 'govuk_app_config', '~> 0.3.0'
 gem 'govuk_sidekiq', '~> 2.0.0'
 gem 'logstasher', '0.4.8'
-gem 'mongoid', '~> 5.0'
-gem 'mongoid_paranoia', '0.2.1'
+gem 'mongoid', '~> 6.2.0'
+gem 'mongoid_paranoia', '0.3.0'
 gem 'nokogiri', '1.6.6.4'
 gem 'plek', '~> 2.0'
-gem 'rack_strip_client_ip', '0.0.1'
-gem 'rails', '4.2.10'
-gem 'state_machines-mongoid', '~> 0.1.1'
+gem 'rack_strip_client_ip', '0.0.2'
+gem 'rails', '5.1.4'
+gem 'rails-controller-testing'
```

Comment:
Why has this gem been added? Could it be explained in the commit message, or ideally, introduced as a separate commit?